inference/server/oasst_inference_server/routes/configs.py

```diff
@@ -66,17 +63,13 @@ class ModelInfo(pydantic.BaseModel):
 ]
 
 
-@router.get("/models")
-async def get_models() -> list[ModelInfo]:
-    allowed_models_list = settings.allowed_models_list
+@router.get("/model_configs")
```

Comment:
could also be `/models/configurations` .. but matter of taste/style

Reply:
I called it "model_config" everywhere else, I felt this was ok